assets/js/base/components/cart-checkout/product-name/index.js

```diff
@@ -9,8 +9,13 @@ import { decodeEntities } from '@wordpress/html-entities';
  */
 import './style.scss';
 
-const ProductName = ( { name, permalink, disabled = false } ) => {
-	return (
+const ProductName = ( {
+	name,
+	permalink,
+	disabled = false,
+	hasLink = true,
```

Comment:
ProductName.propTypes need an update after adding this.

Reply:
Won't be needed since we are now using `disabled`.